datafusion/physical-expr-common/src/aggregate/mod.rs

```diff
@@ -51,6 +51,10 @@ use datafusion_expr::utils::AggregateOrderSensitivity;
 ///
 /// `input_exprs` and `sort_exprs` are used for customizing Accumulator as the arguments in `AccumulatorArgs`,
 /// if you don't need them it is fine to pass empty slice `&[]`.
+///
+/// `is_reversed` is used to indicate whether the aggregation is running in reverse order,
+/// it could be used to hint Accumulator to accumulate in the reversed order,
+/// you can just set to false if you are not reversing expression
 #[allow(clippy::too_many_arguments)]
 pub fn create_aggregate_expr(
```

Comment:
Given the number of arguments it is getting, maybe it would be good to define a builder like the following:

```rust
let builder = AggregateExprBuilder::new(fun),
  .with_input_phyiscal_exprs(input_phy_exprs)
  ...
  .with_name(name)
  .build()?
```

🤔